### PR TITLE
Docs: sphinx conversion - methods chapter remainder

### PR DESCRIPTION
**Pull request by cmills1095**
Cleaning up the rest of the methods chapter

### CODE REVIEW

File: doc/sphinx/user/methods/freesurface/arbitrary-le-implementation.md [hunk 1]
```diff
@@ -21,21 +21,17 @@ the mesh velocity. The mesh velocity is calculated by solving
...
+{math:numref}`eq:temperature-Boussinesq-linear` becomes
 
 ```{math}
-\rho C_p \left(\frac{\partial T}{\partial t} + \left(\mathbf u - \mathbf u_m \right) \cdot\nabla T\right)
-  - \nabla\cdot k\nabla T
-  =
-  \rho H
-   \quad
-   \textrm{in $\Omega$}.
+\rho C_p \left(\frac{\partial T}{\partial t} + \left(\mathbf u - \mathbf u_m \right) \cdot\nabla T\right)- \nabla\cdot k\nabla T=  \rho H   \quad   \textrm{in $\Omega$}.
```
Comment (gassmoeller) [inline doc/sphinx/user/methods/freesurface/arbitrary-le-implementation.md:36]: Can you revert this change? I found the old version much easier to read (plus the new one has some weird spacing for some operators).

File: doc/sphinx/user/methods/melt-transport.md [hunk 2]
```diff
@@ -13,44 +14,24 @@ through and interacting with a viscously deforming host rock. This requires
 the advection of a compositional field representing the volume fraction of
 melt present at any given time (the porosity $\phi$), and also a change of the
 mechanical part of the system. The latter is implemented using the approach of
-(Keller, May, and Kaus 2013) and changes the Stokes system to
+{cite:t}`keller:etal:2013` and changes the Stokes system to
 ```{math}
+:label: eq:stokes-1-melt
+  -\nabla \cdot \left[2\eta \left(\varepsilon(\mathbf{u}_s) - \frac{1}{3}(\nabla \cdot \mathbf{u}_s)\mathbf 1\right)\right] + \nabla p_f + \nabla p_c  = \rho \mathbf g   \qquad   \textrm{in $\Omega$}, \\
```
Comment (gassmoeller) [inline doc/sphinx/user/methods/melt-transport.md:20]: Please keep the formatting of these equation and the ones below. Previously the individual terms were separated so that they were somewhat readable, the long lines make them very hard to read.

File: doc/sphinx/user/methods/melt-transport.md [hunk 3]
```diff
@@ -81,34 +62,22 @@ finally allows us to write
...
-  \\
-  &\quad
-  - \frac{\phi }{\rho_f} \mathbf{u}_s \cdot \nabla\rho_f
-  - (\mathbf{u}_s \cdot \mathbf g ) (1 - \phi) \beta_s \rho_s
-  \notag
-  \\
-  &\quad
-  - K_D \mathbf g \cdot \nabla \rho_f .
-  \notag
+\nabla \cdot \mathbf{u}_s - \nabla \cdot K_D \nabla p_f  - K_D \nabla p_f \cdot \frac{\nabla \rho_f}{\rho_f} &= - \nabla \cdot (K_D\rho_f \mathbf g)  \\
```
Comment (gassmoeller) [inline doc/sphinx/user/methods/melt-transport.md:77]: Same here, please keep the old formatting of the equation.

File: doc/sphinx/user/methods/melt-transport.md [hunk 4]
```diff
@@ -118,47 +87,45 @@ The melt velocity is computed as
...
+:label: eq:porosity
 \begin{align}
-  \label{eq:porosity}
-  \rho_s \frac{\partial (1 - \phi)}{\partial t} + \nabla \cdot \left[ \rho_s (1 - \phi) \mathbf{u}_s \right]
-  &=
-  - \Gamma
-  & \quad
-  & \textrm{in $\Omega$},
-  i=1\ldots C
+  \rho_s \frac{\partial (1 - \phi)}{\partial t} + \nabla \cdot \left[ \rho_s (1 - \phi) \mathbf{u}_s \right]  &=- \Gamma & \quad  & \textrm{in $\Omega$}, i=1\ldots C
```
Comment (gassmoeller) [inline doc/sphinx/user/methods/melt-transport.md:108]: and here

File: doc/sphinx/user/methods/melt-transport.md [hunk 5]
```diff
@@ -118,47 +87,45 @@ The melt velocity is computed as
...
 
 In order to solve this equation in the same way as the other advection
 equations, we replace the second term of the equation by:
 
 ```{math}
-\nabla \cdot \left[ \rho_s (1 - \phi) \mathbf{u}_s \right]
-= \left( 1-\phi \right) \left( \rho_s \nabla \cdot \mathbf{u}_s
-+ \nabla \rho_s \cdot \mathbf{u}_s \right)
-- \nabla \phi \cdot \rho_s \mathbf{u}_s
+\nabla \cdot \left[ \rho_s (1 - \phi) \mathbf{u}_s \right] = \left( 1-\phi \right) \left( \rho_s \nabla \cdot \mathbf{u}_s + \nabla \rho_s \cdot \mathbf{u}_s \right) - \nabla \phi \cdot \rho_s \mathbf{u}_s
```
Comment (gassmoeller) [inline doc/sphinx/user/methods/melt-transport.md:116]: same here, and for the other equations below.

File: doc/sphinx/user/methods/nullspace-removal.md [hunk 6]
```diff
@@ -19,27 +19,45 @@ ASPECT allows the user to specify if their model has a nullspace.
 If so, any power in the nullspace is calculated and removed from the solution after every timestep.
 There are two varieties of nullspace removal implemented: removing net linear/angular momentum, and removing net translations/rotations.
 
-For removing linear momentum we search for a constant velocity vector $\bf c$ such that $$\int_\Omega \rho ({\bf u - c}) = 0$$
+For removing linear momentum we search for a constant velocity vector $\bf c$ such that
```
Comment (gassmoeller) [inline doc/sphinx/user/methods/nullspace-removal.md:22]: This one reads better in the new form. In general the aim is to make the equation readable, both in its source form (in the md file) and when rendered on the website.

File: doc/sphinx/user/methods/nullspace-removal.md [hunk 7]
```diff
@@ -19,27 +19,45 @@ ASPECT allows the user to specify if their model has a nullspace.
...
+```{figure-md} fig:rigid_rotation
+<img src="../../_static/images/rigid_rotation.*" alt="Example of nullspace removal. On the left the nullspace (a rigid rotation) is removed, and the velocity vectors accurately show the mantle flow. On the right there is a significant clockwise rotation to the velocity solution which is making the more interesting flow features difficult to see."  width="80%"/>
+
+Example of nullspace removal. On the left the nullspace (a rigid rotation) is removed, and the velocity vectors accurately show the mantle flow. On the right there is a significant clockwise rotation to the velocity solution which is making the more interesting flow features difficult to see.
+```
+
+:::{admonition} TODO
+:class: error
+
+parameters reference
```
Comment (gassmoeller) [inline doc/sphinx/user/methods/nullspace-removal.md:62]: what does this mean? add to the text

File: doc/sphinx/user/methods/particles.md [hunk 8]
```diff
@@ -5,12 +5,11 @@ ASPECT can, optionally, also deal with
 particles (sometimes called "tracers"). Particles can be thought
 of as point-like objects that are simply advected along with the flow. In
 other words, if $\mathbf u(\mathbf x,t)$ is the flow field that results from
-solving equations {math:numref}`eq:stokes-1-{math:numref}`eq:stokes-2, then the
+solving equations {math:numref}`eq:stokes-1`-{math:numref}`eq:stokes-2`, then the
 $k$th particle's position satisfies the equations
 ```{math}
 \begin{align}
-  \frac{\partial}{\partial t} \mathbf x_k(t)
```
Comment (gassmoeller) [inline doc/sphinx/user/methods/particles.md:12]: again here and below, keep the formatting of equations whenever possible

File: doc/sphinx/user/methods/particles.md [hunk 9]
```diff
@@ -30,21 +29,23 @@ of the $m$th property attached to the $k$th particle, then
...
 input parameters are listed in
-{ref}`parameters:Postprocess/Particles`44]. The implementation of
-particles is discussed in great detail in (Gassm&ouml;ller et al. 2018).
+{ref}`parameters:Postprocess/Particles`. The implementation of
+particles is discussed in great detail in {cite:t}`gassmoller:etal:2018`.
+
+:::{admonition} TODO
+:class: error
+
+Parameter references
```
Comment (gassmoeller) [inline doc/sphinx/user/methods/particles.md:50]: same as above

**Review by gassmoeller — COMMENTED** [review]
Thanks for going through this section! I left some comments, I think there are some changes I would like to see undone, but most of them look good.

File: doc/sphinx/references.bib [hunk 10]
```diff
@@ -296,16 +296,6 @@ @ARTICLE{G09
 }
 
 
-@ARTICLE{GPP11,
```
[the remark this answers is not in the document]
Comment (cmills1095) [reply doc/sphinx/references.bib:299]: Duplicate on line 11259; @article{guermond:etal:2011,
  author =       {{Jean-Luc Guermond and Richard Pasquetti and Bojan Popov}},
  title =        {Entropy viscosity method for nonlinear conservation laws},
  journal =      {Journal of Computational Physics},

File: doc/sphinx/user/install/index.md [hunk 11]
```diff
@@ -65,8 +66,8 @@ The available options can be best presented in form of typical use cases:
     reconfigure underlying libraries or running massively parallel models):
     The most advanced option is to compile and install
     ASPECT from source. This allows maximal control
-    over the underlying libraries like Trilinos
-    and deal.II, as well as easy modifications
+    over the underlying libraries like TRILINOS
+    and DEAL.II, as well as easy modifications
```
[the remark this answers is not in the document]
Comment (cmills1095) [reply doc/sphinx/user/install/index.md:70]: I actually don't remember changing this, must have been set to lowercase on a recent pull request that I hadn't pulled when I submitted mine? Anyways good to know

File: doc/sphinx/user/methods/advection-stabilization/entropy-viscosity.md [hunk 12]
```diff
@@ -47,19 +44,25 @@ approximation is good.
...
 ```{math}
 v_h^\text{max} \vert_K = \alpha_\text{max} h \|\lvert\mathbf u\rvert + \gamma h_K \lvert\varepsilon (\mathbf u)\rvert\|_{\infty,K},
 ```
 where $\gamma\geq 0$ is the aforementioned parameter in front of the strain
 rate.
+
+:::{admonition} TODO
+:class: error
+
+parameter references
```
[the remark this answers is not in the document]
Comment (cmills1095) [reply doc/sphinx/user/methods/advection-stabilization/entropy-viscosity.md:67]: As of now all parameter references don't link to anything. I've been leaving these TODOs in every section that has parameter references so that we can go back and fix things if and when we have something specific to link to

[the remark this answers is not in the document]
Comment (gassmoeller) [reply doc/sphinx/user/methods/advection-stabilization/entropy-viscosity.md:67]: Please replace these comments with explanations that also work for someone else. (e.g. what you wrote in your comment). You never know who will fix these things up later (or if you yourself will remember), so it is important to not rely on your current knowledge when writing the comments.

**Review by gassmoeller — COMMENTED** [review]
Please expand the comment about the parameter reference (you can copy & paste it), afterwards this is ready to go.

**Review by gassmoeller — APPROVED** [review]
